### ISSUE TITLE
Fuzzy Tray font

### ISSUE THREAD

**Issue by eduardomcorrea** [LibreHardwareMonitor/LibreHardwareMonitor#586]
The numbers are fuzzy when using a light taskbar on Windows 11.
Please, add a configuration to enable or disable taskbar font border.

**Comment by peacepenguin** [comment]
Below are my notes on the code locations and examples on where/how to adjust the main tray icon font. I put this together to simply make the font slightly larger on my system.

I'm not sure how to add a font selector dialog to the UI, but you can at least compile your own version with font increased and customized that might alleviate your issue.

Compiling the changes is easy, just get visual studio 2019 community edition (free), and install the desktop app c# bundle when prompted. Then just download the zip of this code repo, make these changes and compile from the visual studio gui. Or you can use the normal git clone workflow too.

This method overrides the positioning of the "bounds" rectangle that gets calculated based on client screen resolution and DPI. I use 100% dpi on my system, and (0, -3) works to center the new value for me. But may need manually adjusted for your system.

I still need to track down where to change the font itself. Currently LibreHM uses the default system font for message boxes, i want to override that as well but haven't figured that out yet.

Let us know if you're successful!


```
## make tray icon text bigger, and bold:

## UI/SensorNotifyIcon.cs   increase base font sizes: (default 12,11,12)  
# line 106:

                case "Segoe UI": baseSize = 15; break;
                case "Tahoma": baseSize = 14; break;
                default: baseSize = 15; break;


## adjust position: (now that we've made the font bigger, it no longer gets centered correctly

## UI/SensorNotifyIcon.cs

# line 201/202: able to swap between these render methods:
			## libre default: not centered once size is increased:
            TextRenderer.DrawText(_graphics, text, small ? _smallFont : _font, bounds, _color, Color.Transparent, TextFormatFlags.HorizontalCenter | TextFormatFlags.VerticalCenter);
            
			## alternative that re-centers the larger font, ( 0, -3) xy coordinate shift.
			_graphics.DrawString(text, small ? _smallFont : _font, _brush, 0, -3);
			


## add Bold directive to the font for further readability improvement:
#line 111

            _font = new Font(family, baseSize * width / 16.0f, FontStyle.Bold, GraphicsUnit.Pixel);
            _smallFont = new Font(family, 0.75f * baseSize * width / 16.0f, GraphicsUnit.Pixel);
```

**Comment by PhyxionNL** [comment]
Should've been fixed by #666, closing...
